cli/rocks/rocks.go

```diff
@@ -0,0 +1,161 @@
+package rocks
+
+import (
+	"embed"
+	"fmt"
+	"os"
+
+	"github.com/tarantool/tt/cli/cmdcontext"
+	lua "github.com/yuin/gopher-lua"
+)
+
+//go:embed extra/*
+//go:embed third_party/luarocks/src/*
+var luarocks embed.FS
+
+func readEmbedFile(f embed.FS, p string) (string, error) {
+	c, err := f.ReadFile(p)
+	if err != nil {
+		return "", err
+	}
+	return string(c), nil
+}
+
+func Exec(cmdCtx *cmdcontext.CmdCtx, args []string) error {
```

Comment:
Write a description for at least the public functions (preferably all of them).

Reply:
Description added